package.json

```diff
@@ -60,7 +59,7 @@
     "react-helmet": "^3.3.2",
     "react-intl": "^2.2.3",
     "react-redux": "^5.0.2",
-    "react-router": "^3.0.2",
+    "react-router-dom": "4.0.0-beta.5",
```

Comment:
this peer dependency needs to be updated to `-beta.6` as well. This devDependency has been updated already.